plugins/woocommerce-blocks/assets/js/blocks/mini-cart/edit.tsx

```diff
@@ -135,6 +151,42 @@ const Edit = ( { attributes, setAttributes }: Props ): ReactElement => {
 							}
 						/>
 					</BaseControl>
+					<BaseControl
+						id="wc-block-mini-cart__product-count-toggle"
+						label={ __( 'Product Count', 'woocommerce' ) }
+					>
+						<SelectControl
```

Comment:
I think it might be nicer to display these options with radio inputs, that way the entire text string for the longer options can be displayed, while in a select it may be cut off. Do you agree?

Reply:
Yes I agree. I also tried it and here's how it will look with a few CSS (gap between the options)
<img width="280" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77401999/c119faf6-547c-4ff9-b8eb-eb91a5edb3b4">

Reply:
> Yes I agree. I also tried it and here's how it will look with a few CSS (gap between the options) <img alt="image" width="280" src="https://private-user-images.githubusercontent.com/77401999/340841427-c119faf6-547c-4ff9-b8eb-eb91a5edb3b4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg4NzM2MDksIm5iZiI6MTcxODg3MzMwOSwicGF0aCI6Ii83NzQwMTk5OS8zNDA4NDE0MjctYzExOWZhZjYtNTQ3Yy00ZmY5LWI4ZWItZWI5MWE1ZWRiM2I0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIwVDA4NDgyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzNmViYzQ3OGE0M2YwZTgzNWUwNmFkMGIxODk3ZjY5YWM2YzcxZGU4NGMxMjVmYzM4N2VlYjEzNTIwYTkyYTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.QpcPVP7ddkms5XPhHs8sUF4tCu4W_nAFvAPQHhbN9nA">
> 
> Should I change it?

I think we should rephrase the wording to make it more user-friendly. 
Something like: 

Display Product Count

⃝ Always (even if 0)
⃝ Never
⃝ Only if cart has any product. 


@nikkivias Could you please help us with some design mocks or text that we should be using to control the product count visibility in Editor? 


The current implementation has the drop-down select option but we are considering changing it to radio buttons.

<img width="673" alt="image" src="https://github.com/woocommerce/woocommerce/assets/11503784/ce4338ea-69b1-40d2-8513-da1725a1b32e">

Reply:
I think going ahead with the shortened versions makes sense here, changing the section title to "Show quantity badge" and the options to:

> ⃝ Always (even if 0)
> ⃝ Never
> ⃝ Only when cart is not empty